app/scripts/models/auth_brokers/base.js

```diff
@@ -175,6 +186,15 @@ define([
 
     _isForceAuthUrl: function () {
       return this.window.location.pathname === '/force_auth';
+    },
+
+    _isSignInUrl: function () {
+      var pathname = this.window.location.pathname;
+      return pathname === '/signin' || pathname === '/oauth/signin';
```

Comment:
It would need a slight refactor: https://github.com/mozilla/fxa-oauth-server/blob/dab5090b42490272bd340b596cc083957affd2e2/lib/routes/redirect.js#L13